data/scripting/win_conditions/hq_hunter.lua

```diff
@@ -0,0 +1,71 @@
+-- =======================================================================
+--                         HQ Hunter Win condition
+-- =======================================================================
+
+include "scripting/coroutine.lua" -- for sleep
+include "scripting/table.lua"
+include "scripting/win_conditions/win_condition_functions.lua"
+
+push_textdomain("win_conditions")
+
+include "scripting/win_conditions/win_condition_texts.lua"
+
+local wc_name = "HQ Hunter"
+-- This needs to be exactly like wc_name, but localized, because wc_name
+-- will be used as the key to fetch the translation in C++
+local wc_descname = _("HQ Hunter")
+local wc_version = 2
+local wc_desc = _ "The tribe or team that can destroy all other headquarters wins the game!. This wincondition does not work with the following starting conditions: Poor Hamlet, Fortified Village, Discovery and New World."
```

Comment:
Better just "works only when you start with Headqueaers", will have to be adopted with every other new win condition.
Could the script start with chehcking if every player has a HQ and send a message if not?

Reply:
well i saw this note as a workaround until we mange to check dependencies of Startconditons with Winconditions in c++.
So I found it better to tell exactly what does not work, cause e.g village or trading outpost or even struggling outpost will work which isn't obvious to the player from their names